test/IDE/module-aliasing-code-complete.swift

```diff
@@ -0,0 +1,128 @@
+/// Test code completion with module aliasing
+/// When -module-alias <alias_name>=<real_name> is applied, code completion should show
+/// the <alias_name> as that's the name which should appear in source files including import statements,
+/// decls, expressions, etc. while getting visible decls come from the module of <real_name>.
+/// Below, XLogging is the alias and mapped to the real name AppleLogging. Note that the real name
+/// AppleLogging should not appear in the code completion results.
+///
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
+
+// RUN: %target-swift-frontend %t/FileLogging.swift -module-name AppleLogging -module-alias XLogging=AppleLogging -emit-module -o %t/AppleLogging.swiftmodule
+
+// RUN: %target-swift-ide-test -code-completion -code-completion-token=MODULE_NAME -source-filename %t/FileLib1.swift -module-alias XLogging=AppleLogging -I %t > %t/result1.txt
+// RUN: %FileCheck %s -check-prefix CHECK1 < %t/result1.txt
+
+// CHECK1-NOT: AppleLogging
```

Comment:
This is only going to check for this string before the start of completion results.  Maybe you want `%FileCheck %s --implicit-check-not=AppleLogging`?

Reply:
Moved CHECK1-NOT after "Begin completion". Lmk if it's still preferable to use ` --implicit-check-not`